parquet-hadoop/src/test/java/org/apache/parquet/statistics/TestStatistics.java

```diff
@@ -19,6 +19,26 @@
 
 package org.apache.parquet.statistics;
 
+import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.BINARY;
```

Comment:
Nit: this makes unnecessary changes to imports and moves static imports above others. Please minimize import changes to make branch management easier.